static-assets/js/tinymce-plugins/ace/plugin.js

```diff
@@ -184,7 +184,12 @@ tinymce.PluginManager.add(PLUGIN_NAME, function (editor, url) {
     aceEditor.on('change', function () {
       editorTextareaEl.value = aceEditor.getValue();
       editor.setContent(aceEditor.getValue());
-      aceModes.inline.getSession().setValue(aceEditor.getValue());
+
+      if (inlineMode) {
+        aceModes.inline.getSession().setValue(aceEditor.getValue());
```

Comment:
Would this path not require the `external_change` to be fired too?

Reply:
Inline mode does not need the event, since that's used on the form, and when saving it retrieves the value and save. For dialog mode it needs to be triggered so the preview control know that there's a change for it to save when focusing out. Though we may leave that event for anyone outside that needs to know about an external change.
Will make the change.